### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_WS- Monitoring_Color Contrast:The color contrast ratio for the headings of the maps are lower than 4.5:1 against the background

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#206]
Environment Details:
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106 
Tool: Color Contrast
MAS ID Violated :MAS1.4.3

**Repro Steps:**
1)Download and install "Microsoft IIS Administration" application.
2)Enable Internet information Services.
3)Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4)Tab till Web Server and press enter
5)Tab till Monitoring and press enter
6)Open Color contrast analyzer
7)Verify color contrast ratio for contents and controls of the page
8)Go to High Contrast Settings (Settings-> Colors-> High Contrast Settings) and check with all the four themes
9)Verify high contrast ratio for contents and controls of the page

**Actual Result:**
The color contrast ratio for the headings ( Requests, Network, Memory, CPU )of the maps are lower than 4.5:1 against the background
  
**Expected Result:**
Color contrast ratio should be >=4.5:1

**User Impact:** 
Visually Impaired Users will face difficulty in navigating if the color contrast ratio comes lower than 4.5:1

**MAS Reference:** 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachments:**
![bug_206_color contrast_issue](https://user-images.githubusercontent.com/42638915/44986370-2202bb80-afa1-11e8-92fc-e03a66870b89.jpg)

**Comment by jhkimnew** [comment]
Fixed with replacing #907c48 with #8b4298.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ColorContrast; #MAS1.4.3; #C+E-H; #.NETWebsites; #Edge

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.iis.net%2F&data=05%7C01%7Cv-bhaprakash%40microsoft.com%7C7a737faa4af14c4ad31a08db634ab100%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638212945750407456%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Q7lvppcGrDLIRxk6EaLiumHk2i6E5S784T%2BniobLEu4%3D&reserved=0) home page will be removed.
